packages/mdc-grid-list/constants.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const strings = {
```

Comment:
Those are good points. On the other hand, having more code spread across more files will make changes a bit less localized. For example, since they're always going to be exposed from the foundations' static getters, that means whenever I need to reference another string / cssClass / number within the foundation, I need to make the change in two files instead of one. 

I'm open to leaving this as-is but maybe we should bring this up during one of our daily syncs regarding 1) if we want a convention here 2) what that convention should be?